src/Streamly/Internal/Network/Socket.hs

```diff
@@ -486,32 +517,40 @@ writeChunksWithBufferOf n h = lpackArraysChunksOf n (writeChunks h)
 -- input elements.
 --
 -- @since 0.7.0
-{-# INLINE putBytesWithBufferOf #-}
-putBytesWithBufferOf :: MonadIO m => Int -> Socket -> SerialT m Word8 -> m ()
-putBytesWithBufferOf n h m = putChunks h $ AS.arraysOf n m
+{-# INLINE putBytesWith #-}
+putBytesWith :: MonadIO m => Int -> Socket -> SerialT m Word8 -> m ()
+putBytesWith n h m = putChunks h $ AS.arraysOf n m
 
 -- | Write a byte stream to a socket. Accumulates the input in chunks of
 -- specified number of bytes before writing.
 --
 -- @since 0.7.0
```

Comment:
This should be since 0.9.0, the same applies to other deprecated functions as well.